index.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="styles.css">
+    <title>Kopie rekupe</title>
+</head>
+<body>
+<a href="https://www.rekupe.com/" target="_blank">Rekupe pagina</a>
+<header class="outer-container">
+    <nav class="navbar">
+        <div >
+            <ul>
+                <!--Let op hier nog verwijzen naar betreffende onderdeel-->
+                <li><a href="#casestudies">Case studies</a></li>
+                <li><a href="#services">Services</a></li>
+                <li><a href="#contactus">Contact Us</a></li>
+            </ul>
+        </div>
+    </nav>
+
+
+        <div class="outer-hero">
+            <div class="hero-section">
+                <h1>Your website is an investment in the success of your business.</h1>
+                <p>We are here to help you succeed.</p>
+                <button type="button" class="cta-btn">Let's work together</button>
+            </div>
+                <object class="sqline" data="assets/squiggle_line_stripe.svg"></object>
+            <div class="skewed">skewed </div>
+        </div>
+</header>
+
+    <main1 class="outer-container-2">
+        <div class="container-0">
+            <h2>Rekupe is a digital agency in Los Angeles.</h2>
+            <h2>We use strategic UX, SEO optimization and beautiful visual designs to help your business grow.
+            </h2>
+            <h2>We see every project as a fresh opportunity, pushing ourselves to create new digital experiences.​
+            </h2>
+            <p>Have a question? <a class="contact" href="#contactus"href="#contactus">Click here</a> to contact us!</p>
+        </div>
+        <div class="skewed2">skewed </div>
+    </main1>
+
+    <main2 class="outer-container-2">
+        <div class="work">
+        <div class="container-1">
+            <div class="laptops">
+                <figure>
```

Comment:
I.p.v een figure zou ik eerder een span aanraden om te gebruiken! Een figure is namelijk self contained content, een span is een inline container voor content, maar er zit “niks” in (behalve als je een image erin stopt). Het verschil is dat hij dan de image als de content ziet (en dus mooier uit zou komen op de website)